pertpy/tools/_distance_tests.py

```diff
@@ -0,0 +1,249 @@
+from __future__ import annotations
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+from rich.progress import track
+from sklearn.metrics import pairwise_distances
+from statsmodels.stats.multitest import multipletests
+
+from ._distances import Distance
+
+
+class DistanceTest:
+    """Runs permutation tests using a distance of choice between groups of cells,
```

Comment:
Must be on a single line. Any detailed explanations should be in a new section below -->

```
def reee():
"""summary.

details bla

Args:
    param
"""
```

roughly